.travis.yml

```diff
@@ -12,6 +12,7 @@ rvm:
   - 2.0
   - 2.1
   - 2.2
+  - 2.3.0
```

Comment:
Travis doesn't detect Ruby from `2.3` string.
Such as https://travis-ci.org/tricknotes/errbit/jobs/99426151

So I specified it as `2.3.0`.